haskell-src/Concordium/Types/Execution.hs

```diff
@@ -715,6 +722,20 @@ data Event =
                -- | The memo.
                tmMemo :: !Memo
            }
+           -- | Contract invocation was interrupted. This only applies to V1 contracts.
+           | Interrupted {
+               -- |Address of the contract that was interrupted.
+               iAddress :: !ContractAddress,
+               -- |Partial event log generated in the execution before the interrupt.
+               iEvents :: ![Wasm.ContractEvent]
+               }
+           -- | Contract execution resumed. This only applies to V1 contracts.
+           | Resumed {
+               -- |Address of the contract that was interrupted.
+               rAddress :: !ContractAddress,
+               -- |Whether the operation succeeded.
```

Comment:
Perhaps something like:
```suggestion
               -- |Whether the operation, which caused the interrupt, succeeded.
```

Reply:
This would be incorrect. The contract invoked an operation, which failed. It is not the initiator of the operation that failed.

Reply:
I added a different clarification.